src/components/common/Application.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState, type FormEvent } from "react";
+import axios from "axios";
+import Select from 'react-select';
+
+const Application = ( 
+  {name}: {name: string}
```

Comment:
we might need one more prop `isRegOpen`. We will not be accepting applications for all cohorts simultaneously

Reply:
can we change to `cohortName`, as name is misleading